cmd/kcp-front-proxy/options/options.go

```diff
@@ -17,95 +17,45 @@ limitations under the License.
 package options
 
 import (
-	"os"
-	"path/filepath"
-
 	"github.com/spf13/pflag"
 
-	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 	"k8s.io/component-base/config"
 	"k8s.io/component-base/logs"
 
 	proxyoptions "github.com/kcp-dev/kcp/pkg/proxy/options"
 )
 
 type Options struct {
-	SecureServing  apiserveroptions.SecureServingOptionsWithLoopback
-	Authentication Authentication
-	Proxy          proxyoptions.Options
-	Logs           *logs.Options
-
-	RootKubeconfig  string
-	RootDirectory   string
-	ProfilerAddress string
+	Proxy proxyoptions.Options
+	Logs  *logs.Options
 }
 
 func NewOptions() *Options {
 	o := &Options{
-		SecureServing:  *apiserveroptions.NewSecureServingOptions().WithLoopback(),
-		Authentication: *NewAuthentication(),
-		Proxy:          *proxyoptions.NewOptions(),
-		Logs:           logs.NewOptions(),
-
-		RootKubeconfig: "",
-		RootDirectory:  ".kcp",
+		Proxy: *proxyoptions.NewOptions(),
+		Logs:  logs.NewOptions(),
 	}
 
 	// Default to -v=2
 	o.Logs.Config.Verbosity = config.VerbosityLevel(2)
-
```

Comment:
Only the Logs option remains now, so we can set this loglevel - I couldn't see a clear pattern of how this is done in other components, should I move it to pkg/proxy too or leave this here (since the log level is really related to the cmd usage)?